src/QuartzNET-DynamoDB/JobStore.cs

```diff
@@ -996,8 +1006,6 @@ public IList<IOperableTrigger> AcquireNextTriggers(DateTimeOffset noLaterThan, i
 
                 // multiple instance management. Create a running scheduler for this instance.
                 CreateOrUpdateCurrentSchedulerInstance();
-
-                DeleteExpiredSchedulers();
```

Comment:
👍 We can do more work here to reduce cost, including introducing smarts around how often we run ResetTriggersAssociatedWithNonActiveSchedulers